README.rst

```diff
@@ -611,6 +617,29 @@ file (or uber jar) containing all of the function dependencies.
 You can find a full Java example in the `samples/java <samples/java>`__
 folder
 
+**.NET Core**
+
+To use SAM Local with compiled languages, such as .NET Core that require a packaged artifact (e.g. a ZIP), you can specify the location of the artifact with the `AWS::Serverless::Function` `CodeUri` property in your SAM template.
+
+For example:
+
+```
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  ExampleJavaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: HelloWorld::HelloWorld.Function::Handler
+      CodeUri: ./artifacts/HelloWorld.zip
+      Runtime: dotnetcore2.0
+```
+
+You should then build your ZIP file using your normal build process.
+
+You can find a .NET Core example in the `sam init` folder.
```

Comment:
Can we change the wording here to be 'You can generate a .NET Core example by `sam init --runtime dotnetcore`'? While it is an example, its specific to cookiecutter and could be hard to parse. Generating the sample seems like a better experience to see and play with.